src/polymer/polymer2-element-scanner.ts

```diff
@@ -188,7 +188,7 @@ export class Polymer2ElementScanner implements JavaScriptScanner {
         description: (docs.description || '').trim(),
         superClass: this._getExtends(node, docs, warnings, document),
         mixins: jsdoc.getMixins(document, node, docs, warnings),
-        privacy: getOrInferPrivacy(className || '', docs, false),
+        privacy: getOrInferPrivacy(className || '', docs, false, 'private'),
```

Comment:
Dunno? There are no existing tests for this. I would say private by default makes more sense since quite often element classes are defined within closures and only publicly available via `document.createElement`. E.g.:

```js
(() => {

class MyElement extends Polymer.Element {
...
}

window.customElements.define(MyElement.is, MyElement);

})();
```
That's just my intuition though. I'm sure you could just as well argue for public by default.